lib/query.js

```diff
@@ -81,7 +81,7 @@ function Query(conditions, options, model, collection) {
 
   if (model) {
     this.model = model;
-    this.schema = model.schema;
```

Comment:
No need to make this change in `query.js`. Queries don't have user-defined properties. We just need to make this change for documents.